src/app/routes/profile/components/profile.js

```diff
@@ -1,75 +1,38 @@
-                                                                <i className="fa fa-envelope" />&nbsp;&nbsp;<a href={"mailto:"+props.user.email}>{props.user.email}</a>
-                                                            </p>
-                                                        </li>
-                                                        <li>
-                                                            <p className="text-muted">
-                                                                <i className="fa fa-phone" />&nbsp;&nbsp;<span className="txt-color-darken">{props.user.phone}</span>
-                                                                <small className="pull-right text-primary">{props.user.phone_type}</small>
-                                                            </p>
-                                                        </li>
-                                                        <li>
-                                                            <p className="text-muted">
-                                                                <i className="fa fa-map-marker" />&nbsp;&nbsp;<span className="txt-color-darken">
-                                                                    {props.user.city + ", " + props.user.state}
-                                                                </span>
-                                                                <small className="pull-right text-primary">{props.user.address_type}</small>
-                                                            </p>
-                                                        </li>
-                                                    </ul>
-                                                    <br />
-                                                    <button onClick={props.editProfileClicked} className="btn btn-default btn-xs"><i className="fa fa-user" /> Edit Information</button>&nbsp;&nbsp;
-                                                    <button className="btn btn-default btn-xs"><i className="fa fa-envelope-o" /> Send Email</button>
-                                                    <br/><br/>
-                                                </div>
-                                            </div>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+        <div id="content" className="animated fadeInDown">
+            <h3 className="text-center text-danger">My Profile</h3><hr/>
+            <div className="col-lg-7 col-md-6 text-center" style={boxShadowStyle}>
+                <br/>
+                <img src="assets/img/avatars/sunny.png" style={bigCirclePhotoStyle} />
+                <h1 className="text-danger"><br/>{props.user.firstname + " "}<span className="semi-bold">{props.user.lastname}</span>
```

Comment:
perhaps we should just map the last name to and so forth to it's own prop. that way this presentational view doesn't have to know about the user structure in the store.